### ISSUE TITLE
itest flake: revoked_uncooperative_close_retribution_zero_value_remote_output

### ISSUE THREAD

**Issue by joostjager** [lightningnetwork/lnd#5645]
```
=== RUN   TestLightningNetworkDaemon/tranche00/47-of-82/bitcoind/revoked_uncooperative_close_retribution_zero_value_remote_output
    test_harness.go:91: Failed: (revoked uncooperative close retribution zero value remote output): exited with error: 
        *errors.errorString unable to find Dave's justice tx in mempool: wanted 1, found 0 txs in mempool: []
        /home/joost/lightninglabs/lnd/lntest/itest/lnd_revocation_test.go:460 (0xf71e1e)
        	testRevokedCloseRetributionZeroValueRemoteOutput: t.Fatalf("unable to find Dave's justice tx in mempool: %v",
        /home/joost/lightninglabs/lnd/lntest/itest/test_harness.go:115 (0xf1b2be)
        	(*harnessTest).RunTestCase: testCase.test(h.lndHarness, h)
        /home/joost/lightninglabs/lnd/lntest/itest/lnd_test.go:247 (0xfbc01b)
        	TestLightningNetworkDaemon.func4: ht.RunTestCase(testCase)
        /usr/local/go/src/testing/testing.go:1050 (0x50cb7c)
        	tRunner: fn(t)
        /usr/local/go/src/runtime/asm_amd64.s:1373 (0x467a81)
        	goexit: BYTE	$0x90	// NOP
```

Can be reproduced by adding a sleep: https://github.com/bottlepay/lnd/tree/itest-repro-revoked

I think the problem is that the test doesn't wait for Dave to process the revocation properly?
https://github.com/bottlepay/lnd/blob/87ff2aed1198dc573bd762f9292cee70c0a4642b/lntest/itest/lnd_revocation_test.go#L441

**Comment by yyforyongyu** [comment]
Should be fixed by #5611

**Comment by yyforyongyu** [comment]
fixed by #6825
